stdlib/2and3/poplib.pyi

```diff
@@ -0,0 +1,68 @@
+# Stubs for poplib (Python 2 and 3)
+
+from mypy_extensions import NoReturn
+import socket
+import ssl
+import sys
+from typing import Any, BinaryIO, Dict, List, Optional, Pattern, Text, Tuple
+
+class error_proto(Exception): pass
+
+POP3_PORT: int
+POP3_SSL_PORT: int
+CR: bytes
+LF: bytes
+CRLF: bytes
+
+
+class POP3:
+    if sys.version_info >= (3, 0):
+        encoding: Text
+
+    host: Text
+    port: int
+    sock: socket.socket
+    file: BinaryIO
+    welcome: bytes
+
+    def __init__(self, host: Text, port: int = ..., timeout: float = ...) -> None: ...
+    def getwelcome(self) -> bytes: ...
+    def set_debuglevel(self, level: int) -> None: ...
+    def user(self, user: Text) -> bytes: ...
+    def pass_(self, pswd: Text) -> bytes: ...
+    def stat(self) -> Tuple[int, int]: ...
+    def list(self, which: Optional[Text] = ...) -> bytes: ...
+    def retr(self, which: Text) -> bytes: ...
```

Comment:
I just found (through testing against some working code) that this is wrong -- it returns a tuple (https://docs.python.org/2/library/poplib.html#poplib.POP3.retr). It also conceivably accepts an int as argument.